src/Raven.Server/Documents/Replication/ChangeVectorExtensions.cs

```diff
@@ -28,6 +29,8 @@ public static string SerializeVector(this ChangeVectorEntry[] self)
             if (self == null)
                 return null;
 
+            self = self.OrderBy(x => x.DbId, StringComparer.OrdinalIgnoreCase).ToArray();
```

Comment:
```suggestion
            Array.Sort(self, (x,y) => string.CompareOrdinal(x.DbId, y.DbId));
```

Let's avoid allocating a new array here each time

Reply:
done